GitCommands/Logging/CommandLog.cs

```diff
@@ -66,7 +66,7 @@ private void LogProcessEnd(int? exitCode = null, Exception? ex = null)
 
     public sealed partial class CommandLogEntry
     {
-        [GeneratedRegex(@"((-c +[^ ]+)|(--no-optional-locks)) *", RegexOptions.ExplicitCapture)]
+        [GeneratedRegex(@"((-c +[^ ""]+(""[^""]*"")?)|(--no-optional-locks)) *", RegexOptions.ExplicitCapture)]
```

Comment:
Sidenote: I fixed the presentation too, in the future PR where this is seen:  https://github.com/gerhardol/gitextensions/blob/tmp/git-coloring/GitUI/Editor/Diff/AnsiEscapeUtilities.cs#L36

We could also require quotes to simplify the regex

I would rather write the regex something like below, will still miss something like '-c "key=\"apa bepa\""' 
```suggestion
        [GeneratedRegex(@"((-c +([^""]+|(""[^""]*"")))|(--no-optional-locks)) *", RegexOptions.ExplicitCapture)]
```

This is just for presentation, so it does not have to be future proof

Reply:
I had to find (and revert) that "fix" in order to take the screenshots.
I do not like tabs. They cause inconsistent empty space - this time in the lower pane of the Git Command Log.

The proposed regex will not work for quotes inside quotes. I do not want to write a regex for nested strings, which might even need to be escaped.

Reply:
Do we have tests which can help us visualise _what_ strings we expect to support and what strings we won't?

Reply:
No, the presentation must be according to what `-c value` allows, which includes spaces
But it is GE that produces these strings, not users

Reply:
Something like the following still fail:
`Default.Add(new GitConfigItem("core.xxx", "\\\"\\\"\""), "diff");`
But it is GE code, no need to make the regex more complicated